src/com/palmergames/bukkit/towny/db/TownyHModFlatFileSource.java

```diff
@@ -285,7 +287,40 @@ public boolean loadTown(Town town) {
 					} catch (NumberFormatException nfe) {
 					} catch (Exception e) {
 					}
-
+				

```

Comment:
This file doesn't need to be touched, no one has any 7 year old hmod databases that haven't already been converted.